canonical-web-and-design/website-boilerplate
# Add Python testing infrastructure

## Done

- Have a `tests` folder
- Have at least one test in the `tests` folder, based on `unittest`, to test that the `/` homepage returns a `200` response
- `./run test` run `lint-python` and `test-python`
- `./run lint-python` run `flake8` and `black --check --line-length 79` against all Python code (including tests)
- `./run test-python` run the `tests`
- Set up CircleCI with a job called `test-python`, which run the tests
- Added codecov integration on CircleCI

## QA

- Run the command `./run test`, it should end correctly.
- Run the command `./run lint-python`, it should end correctly.
- Run the command `./run test-python`, it should end correctly.
